docs/toc.js

```diff
@@ -445,7 +445,114 @@ module.exports = {
       type: 'menu',
       children: [
         {
-          title: '@storybook/blocks',
+          title: 'Main configuration',
+          pathSegment: '',
+          type: 'menu',
+          children: [
```

Comment:
These (along with _every_ list of anything within the API references) are ordered like so:

1. Overview
2. Required properties
3. Optional properties
4. Experimental properties
5. Deprecated properties

And each of those are then ordered alphabetically.